apps/site/src/app/apply/sections/Form/ProfileInformation.tsx

```diff
@@ -0,0 +1,67 @@
+import RequiredAsterisk from "../Components/RequiredAsterisk";
+import styles from "./Form.module.scss";
+
+export default function ProfileInformation() {
+	return (
+		<div className="flex flex-col gap-5 w-11/12">
+			<p className="text-4xl m-0 font-bold text-center max-[700px]:text-3xl">
+				Profile Information
+			</p>
+
+			<div className="flex gap-5 w-full max-[1000px]:flex-col max-[1000px]:items-center">
+				<div className="flex flex-col w-6/12 max-[1000px]:w-full">
+					<label className={`${styles.label}`} htmlFor="linkedin">
+						LinkedIn
+					</label>
+					<input
+						className={`${styles.input}`}
+						type="url"
+						name="linkedin"
+						id="linkedin"
+						placeholder="http://"
+					/>
+				</div>
+				<div className="flex flex-col w-6/12 max-[1000px]:w-full">
+					<label className={`${styles.label}`} htmlFor="portfolio">
+						Portfolio (Github, website, etc.)
+					</label>
+					<input
+						className={`${styles.input}`}
+						type="url"
+						name="portfolio"
+						id="portfolio"
+						placeholder="http://"
+					/>
+				</div>
+			</div>
+
+			<div className="flex gap-5 w-full">
```

Comment:
Probably make these two text boxes for paragraph answers a component. That way we can pass a prop in for the question text instead of hardcoding it and also increase the number of paragraph questions we want in the app.